Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabDomUtils.h

```diff
@@ -36,13 +36,25 @@ namespace AzToolsFramework
             PrefabDomValueReference FindPrefabDomValue(PrefabDomValue& parentValue, const char* valueName);
             PrefabDomValueConstReference FindPrefabDomValue(const PrefabDomValue& parentValue, const char* valueName);
 
+            enum class StoreInstanceFlags : uint8_t
+            {
+                //! No flags used during the call to LoadInstanceFromPrefabDom.
+                None = 0,
+
+                //! By default an instance will be stored with default values. In cases where we want to store less json without defaults
```

Comment:
This is opposite of the behavior of the json serializer in general. It always excludes the defaults and keeps things compact. 

So, this flag should instead be 'StoreDefaultValues' and we should only store defaults if the flag is set

Reply:
Currently discussing this with Ram and Scott privately, will either resolve or fix this depending on what comes up (likely tomorrow).
(Of course everyone else is free to weigh in!)

Reply:
I'm generally a fan of flags, logic, etc, being consistently in the positive voice (it helps avoid double-negatives), so I'm supportive of "StoreDefaultValues" over "DoNotStoreDefaultValues".  It also aligns nicely with the internal implementation of "m_keepDefaults".

Reply:
As discussed elsewhere, we determined to remove the negative but keep the current behavior. So I'm renaming the flag to "StripDefaultValues".